### PR TITLE
Add exploring data example

### PR DESCRIPTION
**Pull request by piotrwitkowski**

### CODE REVIEW

File: examples/ExploringData/Program.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            Origin = "LHR",
+            Destination = "STN",
+            DepartureDate = DateTime.Now.AddMonths(12).ToString("yyyy-MM-dd")
+        }
+    }
+};
+
+Console.WriteLine("Requesting offers");
+var response = await client.OfferRequests.Create(offersRequest, returnOffers: false);
+Console.WriteLine($"Retrieved {response.Offers.Count()} offers");
```
Comment (sgerrand) [inline examples/ExploringData/Program.cs:56]: I don't think that requesting flight offers is related to the other actions performed in this program. Should this section be removed?
```suggestion
```

**Review by sgerrand — COMMENTED** [review]
💭

File: examples/ExploringData/Program.cs (hunk 1 above)
Comment (piotrwitkowski) [reply examples/ExploringData/Program.cs]: Agreed. Fixed

Comment (sgerrand) [reply examples/ExploringData/Program.cs]: Thank you!

**Review by sgerrand — APPROVED** [review]
LGTM. 👍

**Pull request merged**